AWSSDKSwiftCLI/Sources/AWSCLIUtils/String+Utils.swift

```diff
@@ -37,3 +37,9 @@ extension String {
     /// Returns the string that represents a newline
     static var newline: Self { "\n" }
 }
+
+public func printError(_ items: Any..., separator: String = " ", terminator: String = "\n") throws {
```

Comment:
A simple free function that acts like a `print()` statement but writes to stdErr.